eve/main.yml

```diff
@@ -170,6 +170,7 @@ stages:
           # Building the latex twice to catch up the reference
           tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
           tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
+        workdir: docs/
```

Comment:
I was mistaken, this actually needs to be `workdir: build/docs`. I just tried it and was able to build in CI.